src/cs50/sql.py

```diff
@@ -357,6 +365,18 @@ def shutdown_session(exception=None):
                 elif command in ["DELETE", "UPDATE"]:
                     ret = result.rowcount
 
+                # If COMMIT or ROLLBACK, turn on autocommit mode
+                elif command in ["COMMIT", "ROLLBACK"] and "TO" not in statement:
+                    session.close()
+                    self._session = None
+
+
+                # If autocommit is on, commit and close
+                if self._session is None and command not in ["COMMIT", "ROLLBACK"]:
+                    if command not in ["SELECT"]:
```

Comment:
Do we envision other list elements?

Reply:
No, and upon further research this inner condition is probably best off removed altogether -- it may be better to just always commit so as to be sure we're releasing any locks (per https://stackoverflow.com/a/310319/6062385 [explains why commit] and https://stackoverflow.com/a/32394501/6062385 [explains how session.close() works]).

Reply:
Sounds good. (Just asking because `==` would be sufficient otherwise.)